OpenRA.Mods.Common/Traits/Player/BaseAttackNotifier.cs

```diff
@@ -43,13 +43,12 @@ public class BaseAttackNotifier : INotifyDamage
 		readonly RadarPings radarPings;
 		readonly BaseAttackNotifierInfo info;
 
-		int lastAttackTime;
+		long lastAttackTime;
 
 		public BaseAttackNotifier(Actor self, BaseAttackNotifierInfo info)
 		{
 			radarPings = self.World.WorldActor.TraitOrDefault<RadarPings>();
 			this.info = info;
-			lastAttackTime = -info.NotifyInterval * 25;
```

Comment:
We'll still need this (as `lastAttackTime = -info.NotifyInterval`) otherwise the notification won't run within the first `NotifyInterval` after game start.